Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -2351,4 +2354,98 @@ - (void)testModifyCustomHeadersAfterOpeningRealm {
     [self waitForExpectations:@[ex3, ex4] timeout:4.0];
 }
 
+#pragma mark - Read Only
+
+- (RLMSyncUser *)userForTest:(SEL)sel {
```

Comment:
Doesn't this method already exist elsewhere?

Reply:
You'd think so, right? I couldn't find it anywhere though and the existing tests just all duplicate this logic. I skipped updating all the existing tests to use this to simplify merging it to v10.